src/components/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import { Filter } from './Filter';
+import { useTodos } from '../utils/TodoContext';
+
+export const Footer: React.FC = () => {
+  const { todos, removeTodo } = useTodos();
+  const itemsLeft = todos.filter(todo => !todo.completed).length;
```

Comment:
```suggestion
  const todosLeft = todos.filter(todo => !todo.completed).length;
```

not critical but will be informative)